### PR TITLE
Memory leak fix

### PR DESCRIPTION
**Pull request by Hsankesara**

### CODE REVIEW

File: model-invocation-endpoint/mlflow_interface.py [hunk 1]
```diff
@@ -14,14 +14,31 @@
...
 from requests.exceptions import ConnectionError
 from botocore.exceptions import EndpointConnectionError, ClientError
+import subprocess
+from pathlib import Path
+import datetime
 
 class MlflowInterface():
 
+    def clean_tmp(self, path):
+        subprocess.run(["rm", "-r", path])
```
Comment (yatharthranjan) [inline model-invocation-endpoint/mlflow_interface.py:24]: why not use `os.rmdir` ?

Comment (Hsankesara) [reply model-invocation-endpoint/mlflow_interface.py:24]: No reason particularly, the initial patch was using the script and I used subprocess to call that script. Should I change it?

Comment (yatharthranjan) [reply model-invocation-endpoint/mlflow_interface.py:24]: yes, would be consistent as all other code is using python functions. would also make this OS independent (`rm -r` is linux specific and even some versions of linux have different commands like `rmr`)

Comment (yatharthranjan) [reply model-invocation-endpoint/mlflow_interface.py]: ok thanks

**Review by yatharthranjan — APPROVED** [review]
LGTM

**Comment by Hsankesara** [comment]
Thanks, I will update the Rosalind deployment now.

**Pull request merged**